subsys/llext/llext_load.c

```diff
@@ -274,6 +287,16 @@ static int llext_map_sections(struct llext_loader *ldr, struct llext *ext)
 		ldr->sect_map[i].mem_idx = mem_idx;
 		elf_shdr_t *region = ldr->sects + mem_idx;
 
+		/*
+		 * Pre-located ELF objects can have sections for memory regions,
+		 * separate from other sections of the same type. E.g. executable
+		 * sections that will be placed in slower memory. Don't merge such
+		 * sections into main regions
+		 */
+		if (pre_located && shdr->sh_addr < LLEXT_SECTION_MIN_ADDR) {
+			continue;
+		}
```

Comment:
Why do addresses below a certain value get ignored?
I think this is very project-specific. Should we have callback functions to provide this kind of customizations?

Reply:
this is basically a `NULL` area. I use 0 as a base for sections without pre-defined run-time location, but since there can be several such sections, some of them get other low addresses